examples/airflow/README.md

```diff
@@ -1,43 +1,122 @@
-# Debugging [Airflow](https://airflow.apache.org) DAGs with [Marquez](https://marquezproject.github.io/marquez)
+# [Airflow](https://airflow.apache.org) Example
 
-In this example, you'll be introduced to the core concepts of Airflow: _DAGs_, _operators_, and _tasks_. You’ll write your very first DAG and learn how workflows are defined and executed in Airflow using pre-defined datasets and sources.
+In this example, we'll walk you through how to enable an **Airflow DAG** to send lineage metadata to **Marquez**.
 
-## Quickstart
+**What you’ll learn:**
 
-The easiest way to get up and running is with Docker. To start Airflow, run:
+* Collect DAG Metadata with Marquez
+* Explore inter-DAG dependencies using Marquez
+
+> **Note:** We've added [`marquez-airflow`](https://github.com/MarquezProject/marquez/tree/main/integrations/airflow) to `requirements.txt` to automatically send DAG metadata to Marquez.
+
+## Step 1: Start Airflow
+
+To start Airflow, run:
 
 ```bash
 $ ./docker/up.sh
 ```
 
 > **Tip:** Use the `--pull` to pull a tagged image.
 
-Airflow listens on port `8080`. To view the Airflow UI and verify it's running, open http://localhost:8080. Then, browse to http://localhost:3000 to begin exploring DAG metadata via the Marquez UI.
-
-# Airflow
-
-[Airflow](https://airflow.apache.org) is a platform to programmatically author, schedule, and monitor workflows. Below we define some general concepts, but we encourage you to review [Airflow's concept doc](https://airflow.apache.org/docs/stable/concepts.html). 
-
-### DAGs
-
-A `DAG` (or [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) is a collection of `Tasks`. It represents an entire workflow by describing the relationships and dependencies between tasks.
-
-### Tasks
-
-A `Task` represents a unit of work within a workflow. Each task is an implementation of an `Operator`, for example, a [PostgresOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/postgres_operator/index.html#airflow.operators.postgres_operator.PostgresOperator) to execute a SQL query, or a [PythonOperator](https://airflow.apache.org/docs/stable/_api/airflow/operators/python_operator/index.html#airflow.operators.python_operator.PythonOperator) to execute a Python function.
-
-### Operators
-
-While `DAGs` describe how to run a workflow, `Operators` determine what actually gets done by a task.
-
-# UI
+**The `docker/up.sh` will:**
+
+* Start Airflow and install `marquez-airflow` used to collect DAG metadata
+* Start Marquez
+* Start Postgres
+
+To view the Airflow UI and verify it's running, open http://localhost:8080. When all _enabled_ DAGs complete successfully, browse to http://localhost:3000 to begin exploring DAG metadata via the Marquez UI. You should see the following lineage graph for [`etl_orders_7_days`](https://github.com/MarquezProject/marquez/blob/main/examples/airflow/dags/etl_orders_7_days.py):
```

Comment:
I did this step and I don't see anything for etl_orders_7_days

Reply:
I would suggest avoiding using the seed data here because it may confuse people here. If we do, it needs to be explicitly called out in the workflow of its effects and what we are trying to accomplish (e.g. adding a new job to this mock data pipeline system).

Reply:
Is this suppose to be the `./docker/up` in the airflow directory? We have one in the root project named the same thing...